app/prj.conf

```diff
@@ -17,6 +17,9 @@ CONFIG_BUILD_OUTPUT_STRIPPED=y
 # default value.
 CONFIG_CLEANUP_INTERMEDIATE_FILES=n
 
+# Generate zephyr.lst file
+CONFIG_OUTPUT_DISASSEMBLY=y
```

Comment:
what's the benefit of output lst file during build? who will use it? it will somehow make build slower.

Reply:
xtensa-build-zephyr.py will package this file to a compressed file for debug, this file is very useful for debug.  It seems without this file, xtensa-build-zephyr.py will complain.